src/client/dfuse/dfuse_main.c

```diff
@@ -251,70 +251,104 @@ static void
 show_help(char *name)
 {
 	printf(
-	    "usage: %s <mountpoint> [pool] [container]\n"
+	    "usage: %s [OPTIONS] [mountpoint [pool container]]\n"
+	    "\n"
 	    "Options:\n"
 	    "\n"
-	    "	-m --mountpoint=<path>	Mount point to use\n"
+	    "	-m --mountpoint=<path>	Mount point to use (deprecated, use positional argument)\n"
 	    "\n"
 	    "	   --pool=name		pool UUID/label\n"
 	    "	   --container=name	container UUID/label\n"
 	    "	   --path=<path>	Path to load UNS pool/container data\n"
 	    "	   --sys-name=STR	DAOS system name context for servers\n"
 	    "\n"
 	    "	-S --singlethread	Single threaded\n"
-	    "	-t --thread-count=count	Number of threads to use\n"
-	    "   -e --eq-count=count     Number of event queues to use\n"
+	    "	-t --thread-count=count	Total number of threads to use\n"
+	    "	-e --eq-count=count	Number of event queues to use\n"
 	    "	-f --foreground		Run in foreground\n"
 	    "	   --enable-caching	Enable all caching (default)\n"
 	    "	   --enable-wb-cache	Use write-back cache rather than write-through (default)\n"
 	    "	   --disable-caching	Disable all caching\n"
 	    "	   --disable-wb-cache	Use write-through rather than write-back cache\n"
 	    "	-o options		mount style options string\n"
 	    "\n"
-	    "      --multi-user		Run dfuse in multi user mode\n"
+	    "	   --multi-user		Run dfuse in multi user mode\n"
 	    "\n"
 	    "	-h --help		Show this help\n"
 	    "	-v --version		Show version\n"
 	    "\n"
-	    "Specifying pool and container are optional. If not set then dfuse can connect to\n"
-	    "many using the uuids as leading components of the path.\n"
-	    "Pools and containers can be specified using either uuids or labels.\n"
+	    "dfuse performs a user space mount of a DAOS POSIX container at the mountpoint\n"
+	    "directory that is specified as the first positional argument. This directory\n"
+	    "has to exist and has to be accessible to the user, or the mount will fail.\n"
+	    "Alternatively, the mountpoint directory can also be specified with the -m or\n"
+	    "--mountpoint= option but this usage is deprecated.\n"
 	    "\n"
-	    "The path option can be use to set a filesystem path from which Namespace attributes\n"
-	    "will be loaded, or if path is not set then the mount directory will also be\n"
-	    "checked.  Only one way of setting pool and container data should be used.\n"
+	    "The DAOS pool and container can be specified in several different ways"
+	    "(only one way of specifying the pool and container should be used):\n"
+	    "* The DAOS pool and container can be explicitly specified on the command line\n"
+	    "  as positional arguments, using either UUIDs or labels.\n"
```

Comment:
I think it would be worth adding that this is the most common way. A user knows a pool and has created a posix container and can just mount those using dfuse. I think 98% of users will be in this bucket.

Reply:
done